RELEASE-NOTES.md

```diff
@@ -1,4 +1,85 @@
 # Prefect Release Notes
+## Release 2.8.3
+
+### `on_completion` and `on_failure` hooks for flows and tasks
+With this release you can now add client-side hooks to your flows and tasks that will be called when the flow or task enters either a `Completed` or `Failed` state. This is great for any case where you might want to execute code without involvement of the Prefect API. 
+
+Both flows and tasks will accept the arguments `on_completion` and `on_failure` that take a list of callables. These callables will need to accept 3 arguments:
+- `flow`, `flow_run`, and `state` in the case of a flow hook
+- `task`, `task_run`, and `state` in the case of a task hook
+
+Here is an example showing how completion hooks can be added to a flow and a task:
+```python
+from prefect import task, flow
+
+def my_completion_task_hook_1(task, task_run, state):
+    print("This is the first hook - Task completed!!!")
+    
+def my_completion_task_hook_2(task, task_run, state):
+  print("This is the second hook - Task completed!!!")
+    
+def my_completion_flow_hook(flow, flow_run, state):
+    print("Flow completed!!!")
+    
+@task(on_completion=[my_completion_task_hook_1, my_completion_task_hook_2])
+def my_task():
+    print("This is the task!")
+
+@flow(on_completion=[my_completion_flow_hook])
+def my_flow():
+    my_task()
+
+if __name__ == "__main__":
+    my_flow()
+```
```

Comment:
Try to include spacing between code fences and content for readability.

Reply:
Cool - I wasn't certain where the line between proper spacing and not making the example part of the release notes too long was.

Reply:
I don't think it matters how long the markdown is. I try not to have excessive spacing in the code itself though.